vars/pipelineVegavisorManageNode.groovy

```diff
@@ -186,11 +215,12 @@ void call() {
                                         cp ./bin/visor ./ansible/roles/barenode/files/bin/
                                     """
                                 }
+
                                 // create json with function instead of manual
                                 ANSIBLE_VARS = writeJSON(
                                     returnText: true,
                                     json: ANSIBLE_VARS_DICT + [
-                                        release_version: params.RELEASE_VERSION,
+                                        release_version: (params.RELEASE_VERSION.length() > 0 ? params.RELEASE_VERSION : VEGA_VERSION_FROM_STATISTICS),
```

Comment:
I think just doing:
`params.RELEASE_VERSION ?: VEGA_VERSION_FROM_STATISTICS`
should be good.